cmake/Modules/FindFLANN.cmake

```diff
@@ -37,6 +37,7 @@
 # Early return if FLANN target is already defined. This makes it safe to run
 # this script multiple times.
 if(TARGET FLANN::FLANN)
+  set(FLANN_FOUND ON)
```

Comment:
I would set them to True (think is normally how its set, but in the end the effect should be the same 😄  )

Reply:
I did it like in other parts of the code: https://github.com/PointCloudLibrary/pcl/blob/master/cmake/Modules/FindFLANN.cmake#L48 and https://github.com/PointCloudLibrary/pcl/blob/master/cmake/Modules/FindQhull.cmake#L49

Reply:
If you try look in FIND_PACKAGE_HANDLE_STANDARD_ARGS which is in FindPackageHandleStandardArgs.cmake, then they set it to true, but since its seems common to use ON, we can continue with that 👍